### DISCUSSION TITLE
Website from our GitHub repo?

### DISCUSSION THREAD

**Discussion by annefou** [destination-earth/DestinE_ESA_GFTS#2] [Ideas]
Does it make sense to generate HTML pages from our GitHub repository; for the documentation, examples, etc.? For instance, using sphinx or Jupyter book or whatever you think is more appropriate.

**Comment by yellowcap** [comment]
Yes I like the idea. I have used sphinx before, but I heard that Jupyter book is easier to use. So I vote for that one.
